server.js

```diff
@@ -1,25 +1,42 @@
-import express from 'express';
-import dataUriToBuffer from 'data-uri-to-buffer';
-import bodyParser from 'body-parser';
-import svgToDataURL from 'svg-to-dataurl';
+import express from "express";
+import dataUriToBuffer from "data-uri-to-buffer";
+import bodyParser from "body-parser";
+import svgToDataURL from "svg-to-dataurl";
 import { svgo } from "../SvgBackend/src/svgo.config";
+import {OPTIMIZE_SVG,SVG_CODE_TO_BE_INSERTED} from "./src/constants/constants";
 import SVGO from "svgo";
-import cors from 'cors';
+import cors from "cors";
 
 const app = express();
 const port = process.env.PORT || 3500;
 const svgoObject = new SVGO(svgo);
-
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(cors());
 
-app.post('/optimizeSvg',function(req,res){
-    svgoObject.optimize(dataUriToBuffer(req.body.dataUrl).toString())
+app.post(OPTIMIZE_SVG, function(req, res) {
+  svgoObject
+    .optimize(dataUriToBuffer(req.body.dataUrl).toString())
     .then(result => {
-        res.send(JSON.stringify({urlData:svgToDataURL(result.data)}));
+      var newDataWithColorClass = " ";
+      if (result.data.toString().search(SVG_CODE_TO_BE_INSERTED) < 0) {
+        const PositionToInsert = result.data.toString().indexOf(">") + 1;
+        newDataWithColorClass = [
+          result.data.toString().slice(0, PositionToInsert),
+          SVG_CODE_TO_BE_INSERTED,
+          result.data
+            .toString()
+            .slice(PositionToInsert, result.data.toString().length)
+        ].join();
+      } else {
+        newDataWithColorClass = result.data;
```

Comment:
Can you create a utils folder with a file called svgHelper and add this logic into a function called declareColourClass then import that function into this file and call it in this method

Reply:
Done deal.